### DISCUSSION TITLE
structre feature missing

### DISCUSSION THREAD

**Discussion by Tariya321** [James-Yu/LaTeX-Workshop#4108] [Q&A] [answered]
**Problem discription**
It seems like I clicked to cancel the option 'structure', but I cannot find it for anymore

**Apperence**
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/108873513/a9bd5d71-1dcb-4132-a24b-4f3958f42eb1)

**Comment by James-Yu** [comment]
Not extension related.

**Comment by Tariya321** [comment, marked as answer]
Well, I got it 😃 
This is a feature update at VSCide Ver1.64, which support `move views to <locaiton>`.
And I accidently `move <structure> to <secondary side bar>`.

Provided that anyone encounter this problem again, you can find it in `view -> Apperence -> Secondary side bar`
Thanks anyway 👍 👍 👍
